### PR TITLE
Homework for lesson 3

### PR DESCRIPTION
**Pull request by dimloko**

### CODE REVIEW

File: third-lesson/src/metod_task8.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*Имеется целове число (задать с помощью Random rand = new Random(); int x =rand.nextInt() ).
+Это число – количесво денег в рублях. Вывести это число, добавив к нему слово «рублей» в правильном падеже.*/
+
+import java.util.Random;
+
+public class metod_task8 {
+    public static void main(String[] args) {
```
Comment (RomanKashenok) [inline third-lesson/src/metod_task8.java:7]: 12 will be printed incorrectly

**Review by RomanKashenok — APPROVED** [review]
Look at comment please

File: third-lesson/src/metod_task10.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/*Посчитать факториал числа в границах от 10 до 15 (не используя рекурсию).*/
+
+import java.util.Random;
+
+public class metod_task10 {
```
Comment (TheFallen058) [inline third-lesson/src/metod_task10.java:5]: Best practice: Class name = CamalCase. So please start class name from capital later
Class/Interface from capital later
method/variable from lowercase letter
and don't use snake case for class names. It's more common for script languages or sql

File: third-lesson/src/task1.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/*Начав тренировки, спортсмен в первый день пробежал 10 км.
+Каждый день он увеличивал дневную норму на 10% нормы предыдущего дня.
+Какой суммарный путь пробежит спортсмен за 7 дней?*/
+
+public class task1 {
+    public static void main(String[] args) {
+
+        double way = 10.0; //Суммарный путь, 10 - путь за первый день
+        double distance = way;//Расстояние пройденное за день
+        int day = 0;
```
Comment (TheFallen058) [inline third-lesson/src/task1.java:10]: Days start from 1

File: third-lesson/src/task1.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+        //Подсчёт суммарного пути за 6 оставшихся дней
+        while (day < 6) {
+            distance *= 1.1; //Увеличение расстояния за день на 10%
+            way += distance; //увеличение суммарного пути
+            day++;
+        }
+
+        //вывод резульатата
+
+        System.out.printf("Суммарный путь за %d дней %.3f км%n", day + 1, way);
```
Comment (TheFallen058) [inline third-lesson/src/task1.java:21]: Great that result is universal so we can just change number of days!

File: third-lesson/src/task2.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        System.out.printf("|------|-------|%n");
+        System.out.printf("| Часы | Амебы |%n");
+        System.out.printf("|------|-------|%n");
+
+        //Подсчёт колличества амеб, часов и вывод таблицы
+        while (hour < 24) {
+            hour += 3;
+            ameba *= 2;
+            System.out.printf("|%6d| %6d|%n", hour, ameba);
+            System.out.printf("|------|-------|%n");
```
Comment (TheFallen058) [inline third-lesson/src/task2.java:20]: Amazing result view!

File: third-lesson/src/task6.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+/*Напишите программу вывода всех четных чисел от 2 до 100 включительно*/
+
+public class task6 {
+    public static void main(String[] args) {
+        for (int i = 2; i <= 100; i += 2) {
```
Comment (TheFallen058) [inline third-lesson/src/task6.java:5]: Good.
As an option we can use regular counter i++ but check if i%2==0
But your way is faster

File: third-lesson/src/metod_task9.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
+/*Изменить пример с суммой чисел таким образом, чтобы рассчитывалась не сумма, а
+произведение, т.е. факториал числа.*/
+
+public class metod_task9 {
+    public static void main(String[] args) {
+        double averageFact = 0; //Средний факториал, чушь конечно но я так вижу)
+        int n = 0; //Счётчик колличества итераций цикла while для вычисления среднего
+        long fact = 1, sumfact = 0; //Для подсчёта факториала в текущей итерации цикла while и суммы факториалов за все предыдущие итерации
```
Comment (TheFallen058) [inline third-lesson/src/metod_task9.java:8]: good names of variables except sumfact. pay more attention to IDEAs hints

File: third-lesson/src/metod_task13.java [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+import java.util.Scanner;
+
+public class metod_task13 {
+    public static void main(String[] args) {
+        Random rand = new Random();
+        Scanner sc = new Scanner(System.in);
+        int i_max = 0;//Индекс максимального элемента массива
+
+        //Ввод размерности массива
+        System.out.println("Введите размер массива:");
```
Comment (TheFallen058) [inline third-lesson/src/metod_task13.java:13]: no check for incorrect array size. will be great to check then input is correct

File: third-lesson/src/metod_task17.java [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+            }
+        }
+
+        //Копируем верхний треугольник из единиц в зеркальном отражении в нижний треугольник
+        for (int j = 0; j < array.length; j++) {
+            for (int i = 0; i < array.length / 2; i++) {
+                array[array.length - 1 - i][j] = array[i][j];
+            }
+        }
+
```
Comment (TheFallen058) [inline third-lesson/src/metod_task17.java:38]: You code works as expected.
My solution will be something like this:
        for (int i = 0; i < array.length; i++) {
            for (int j = 0; j < array.length; j++) {
                if (i < array.length / 2) {
                    if (i <= j && array.length - i > j) {
                        array[i][j] = 1;
                    }
                } else {
                    if (i >= j && array.length - i < j + 2) {
                        array[i][j] = 1;
                    }
                }
            }
        }
please try to analyze. it uses twice less for circles.

**Review by TheFallen058 — COMMENTED** [review]
Some comments was added, in general good job!

**Pull request merged**